test/stream-muxers.node.js

```diff
@@ -149,5 +150,22 @@ describe('Stream Multiplexing', () => {
         }, 500)
       })
     })
+
+    it('should fail graciously on dead stream mux', function (done) {
+      if (sm.multicodec === '/spdy/3.1.0') {
+        this.skip()
+      }
```

Comment:
No skipping for SPDY

Reply:
Skipping because the behaviour is not consistent across muxers - spdy doesn't return any errors on abrupt termination.